renovate.json

```diff
@@ -2,5 +2,23 @@
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     "config:recommended"
+  ],
+  "labels": ["dependencies"],
+  "baseBranches": ["/core\\d\\d-\\d/"],
```

Comment:
@lengau I have no idea if this actually works (I think Renovate is very confusing). The intention is to have Renovate look at `rockcraft.yaml` on the `coreXY-Z` branches and update the minor and patch versions whenever a new Snapcraft release is made (via github). Does this look OK?